hnn_qt5.py

```diff
@@ -136,7 +149,7 @@ def updatebaseparamwin (self, d):
     self.prmComm.psig.emit(d)
 
   def updatedrawerr (self):
-    self.canvComm.csig.emit(False) # False means do not recalculate error
+    self.canvComm.csig.emit(True) # False means do not recalculate error
```

Comment:
This signal gets emitted when there are new results, so it seems we always want RMSE to be updated

Reply:
I think it was set to False, because in old version of optrun the error was first calculated (synchronously, wait for it), and then signal emitted to redraw it (asynchronously, but don't hold up optimization for redraw).